### PR TITLE
packet integration

### PR DESCRIPTION
**Pull request by will-chan**
- also fixes create_account.sh to include new project requirements.
- including latest go-rancher client.

### CODE REVIEW

File: tools/1 [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```
Comment (cjellick) [inline tools/1]: is this entire file a mistake?

**Comment by cjellick** [comment]
@will-chan you want to wait to merge this until we merge rancher-client and update godeps, right?

**Comment by cjellick** [comment]
@will-chan Are you going to add a packet-specifc test like we have for all the other "first class" drivers?
Also, I think we should have at least a basic validation test for packet prior to merging everything, so that we can prove it works.

**Comment by cjellick** [comment]
Darn it. accidentally clicked the close and comment button

**Comment by will-chan** [comment]
I'll add the packet test.  Actually, after adding it, I realize we should refactor that a bit instead of all this mostly boiler plate code I am adding.

**Pull request merged**